src/layouts/MenuBar.vue

```diff
@@ -51,7 +57,7 @@ export default defineComponent({
 	emits: {
 		toggle: null
 	},
```

Comment:
See there are no other events that need validation, the shorthand array form is preferable.
```suggestion
	emits: ["toggle"],
```